File: Python Excel Examples/CellsApiDemo/row/deleteRows.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "DeleteRows_1.xlsx"
sheetName = "Sheet4"
startRow = 2
total_rows = 2
storage = ""
folder = "/Cells/Row/"
api.delete_rows(name, sheet_name=sheetName, startrow=startRow, total_rows=total_rows,folder=folder, storage=storage)

File: PHP Excel Examples/CellsApiDemo/Row/deleteRows.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "DeleteRows.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet4";
$rowIndex = 2;
$totalRows = 2; //Default value : 1
$apiInstance->deleteRows($name, $sheetName, $rowIndex, $totalRows, $folder, $storage);
